cmd/aro/update_ocp_versions.go

```diff
@@ -34,57 +39,111 @@ func getInstallerImageDigests(envKey string) (map[string]string, error) {
 	if !env.IsLocalDevelopmentMode() {
 		jsonData, err = base64.StdEncoding.DecodeString(string(jsonData))
 		if err != nil {
-			return nil, fmt.Errorf("%s: Failed to decode base64: %v", envKey, err)
+			return fmt.Errorf("%s: Failed to decode base64: %v", envKey, err)
 		}
 	}
 
-	if err = json.Unmarshal(jsonData, &installerImageDigests); err != nil {
-		return nil, fmt.Errorf("%s: Failed to parse JSON: %v", envKey, err)
+	if err = json.Unmarshal(jsonData, envData); err != nil {
```

Comment:
Looking another time, `data` feels a little too generic... Can we make this something like `envInstallerImageDigests` or similar, or is it too late?

Reply:
It's meant to be generic because it's not specific to installer image digests.

I guess it's easy to miss in the diff, but this logic is now part of a helper function named `getEnvironmentData` that's used for both installer image digests and openshift versions.